pyproject.toml

```diff
@@ -12,19 +12,44 @@ packages = [
 python = ">=3.8,<4"
 django = ">=3.2"
 ruff = "^0.0.292"
+typing-extensions = {version = ">=4.0.1", python = "3.11"}
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 pdbpp = "*"
 parameterized = "*"
 pytest-cov = "*"
 pytest = "*"
 pytest-sugar = "*"
 pytest-django = "*"
-django-types = "*"
+django-stubs = "^4.2.7"
+mypy = "^1.8.0"
 
 [tool.ruff]
 line-length = 100
 target-version = "py38"
 ignore = [
     "E501",   # line length is handled by formatter
 ]
+
+[tool.mypy]
```

Comment:
oh, i see where's mypy config. Nice, let's add some more options here. No need for separate file.